.travis.yml

```diff
@@ -7,7 +7,7 @@ git:
      - git submodule update --init --recursive
 language: node_js
 node_js:
-  - node
+  - 10
```

Comment:
Is there any reason for choosing 10 instead of a more recent version?

Reply:
12 would get us ES2017 features :raised_eyebrow:

Reply:
Newer versions of node crash Gulp, as far as I found. Updating Gulp to latest seems non-trivial. Upstream already uses a newer version of Gulp, and we should probably pull that work in if we want to use a newer Gulp.